addons/website/static/src/js/menu/navbar.js

```diff
@@ -37,6 +38,12 @@ var WebsiteNavbar = publicWidget.RootWidget.extend({
      */
     start: function () {
         var self = this;
+            dom.initAutoMoreMenu(this.$('ul.o_menu_sections'), {
+                maxWidth : function () {
+                    // NOTE : navbar will contain different elements in community and enterprise, so we check for both of them here only
```

Comment:
No need of starting a comment with `NOTE : `, all comments are notes ;)
Just put `// The navbar contains different elements in community and enterprise, so we check for both of them here only`